app/controllers/content_types/school_home_pages_controller.rb

```diff
@@ -33,7 +33,7 @@ def slideshow
         'COMMON STYLES' => cascade_block('_cascade/blocks/html/common_styles'),
         'COMMON SCRIPTS' => cascade_block('_cascade/blocks/html/common_scripts'),
         'CUSTOM THEME' => cascade_block('_cascade/blocks/custom_themes/law'),
-        'GLOBAL FOOTER' => cascade_block('_cascade/blocks/html/law_footer'),
+        'GLOBAL FOOTER' => cascade_block('widgets/shared/law_footer'),
```

Comment:
It's just that the way cascade-assets is set up right now, everything is under `widgets`. I personally think at the moment, the law header and footer don't make sense under `_cascade/blocks/html` and for me it would be an unintuitive place to look for it given where the footer and other page elements are located hierarchically in cascade-assets.